src/modules/components/proposal/proposalActions/actions/proposalActionTokenMint/proposalActionTokenMint.test.tsx

```diff
@@ -0,0 +1,118 @@
+import { render, screen } from '@testing-library/react';
+import { modulesCopy } from '../../../../../assets';
+import { OdsModulesProvider } from '../../../../odsModulesProvider';
+import { type IProposalActionTokenMint } from '../../proposalActionsTypes/proposalActionTokenMint';
```

Comment:
```suggestion
import { type IProposalActionTokenMint } from '../../proposalActionsTypes';
```

Will need to make sure is barreled from the index export as well in types.